packages/vite/rollup.config.ts

```diff
@@ -329,10 +329,10 @@ function bundleSizeLimit(limit: number): Plugin {
           .join(''),
         'utf-8',
       )
-      const kb = size / 1024
+      const kb = size / 1000
```

Comment:
Oh this change triggered the limit 😅 
```
packages/vite build: [!] (plugin bundle-limit) Error: Bundle size exceeded 120 kB, current size is 121.86kb.
packages/vite build:     at Object.generateBundle (file:///home/runner/work/vite/vite/packages/vite/rollup.config-1691641081524.mjs:360:23)
packages/vite build:     at /home/runner/work/vite/vite/node_modules/.pnpm/rollup@3.28.0/node_modules/rollup/dist/shared/rollup.js:1909:40
packages/vite build:  ELIFECYCLE  Command failed with exit code 1.
packages/vite build: ERROR: "build-bundle" exited with 1.
packages/vite build: Failed
```

Reply:
The previous limit was actually [120 **kiB**](https://github.com/vitejs/vite/blob/6cb397fd2de29a57ba67962b7865ff310aed4d70/packages/vite/rollup.config.ts#L198-L198), which is 122880 bytes = 122.88 kB

I think 123 kB seems reasonable approximation assuming we do not want to put weird number `122.88` in the config. I did the change in 3b4904f10cea9eb8042e7a57fa23bf8d0c2de319